tests/app/service/test_rest.py

```diff
@@ -3382,6 +3382,20 @@ def test_get_monthly_notification_data_by_service(mocker, admin_request):
     assert response == []
 
 
+@freeze_time('2019-12-11 13:30')
+def test_get_returned_letter_statistics(admin_request, sample_service):
```

Comment:
Should probably do a test for the case were there are no recent returned letters too please

Reply:
Done in 02c824a